staging/src/k8s.io/apiserver/pkg/util/flowcontrol/fq/util.go

```diff
@@ -0,0 +1,8 @@
+package fq
+
+func min(a, b float64) float64 {
```

Comment:
isn't this the same as `math.Min`?

Reply:
Yes, good catch.  This originally was an int which is why this is like that.  I have removed this fxn and fq/util.go from the PR now.